Build failure: pulseeffects-legacy

## Steps To Reproduce

Steps to reproduce the behavior:

1. `git clone --depth=1 https://github.com/nixos/nixpkgs`
2. `cd nixpkgs`
3. `nix-build -A pulseeffects-legacy`

## Build log

<details>

<summary>Build Log</summary>

```
Running phase: unpackPhase
unpacking source archive /nix/store/xh1j8vrl5lwzyiwnjdyb3l2b6sxl47h9-source
source root is source
Running phase: patchPhase
patching script interpreter paths in meson_post_install.py
meson_post_install.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/c9m6yd8fg1flz2j5r4bif1ib5j20a0cy-python3-3.12.8/bin/python3"
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
mesonConfigurePhase flags: --prefix=/nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7 --libdir=/nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/lib --libexecdir=/nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/libexec --bindir=/nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/bin --sbindir=/nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/sbin --includedir=/nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/include --mandir=/nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/share/man --infodir=/nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/share/info --localedir=/nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/share/locale -Dauto_features=enabled -Dwrap_mode=nodownload --buildtype=plain
The Meson build system
Version: 1.6.0
Source dir: /build/source
Build dir: /build/source/build
Build type: native build
Project name: pulseeffects
Project version: 4.8.7
C compiler for the host machine: gcc (gcc 14.2.1 "gcc (GCC) 14.2.1 20241116")
C linker for the host machine: gcc ld.bfd 2.43.1
C++ compiler for the host machine: g++ (gcc 14.2.1 "g++ (GCC) 14.2.1 20241116")
C++ linker for the host machine: g++ ld.bfd 2.43.1
Host machine cpu family: x86_64
Host machine cpu: x86_64
Configuring config.h using configuration
Program msgfmt found: YES (/nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgfmt)
Program appstream-util found: NO
Program desktop-file-validate found: YES (/nix/store/1a68w7414qcrvh3dkmyk434jckrzg41s-desktop-file-utils-0.28/bin/desktop-file-validate)
Configuring about.glade using configuration
Found pkg-config: YES (/nix/store/0d4m43yp69lrm8imxbqgl9zxjwwz52jw-pkg-config-wrapper-0.29.2/bin/pkg-config) 0.29.2
Build-time dependency gio-2.0 found: YES 2.82.1
Program /nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/bin/glib-compile-resources found: YES (/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/bin/glib-compile-resources)
Configuring com.github.wwmm.pulseeffects.service using configuration
Program msginit found: YES (/nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msginit)
Program msgmerge found: YES (/nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgmerge)
Program xgettext found: YES (/nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/xgettext)
Program itstool found: YES (/nix/store/j393h0qyaxs58fl4ppmckgv7dsilzd9k-itstool-2.0.7/bin/itstool)
Program itstool found: YES (/nix/store/j393h0qyaxs58fl4ppmckgv7dsilzd9k-itstool-2.0.7/bin/itstool)
Program msgmerge found: YES (/nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgmerge)
Program msgfmt found: YES (/nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgfmt)
Run-time dependency libpulse found: YES 17.0
Run-time dependency glib-2.0 found: YES 2.82.1
Run-time dependency glibmm-2.4 found: YES 2.66.7
Run-time dependency gtk+-3.0 found: YES 3.24.43
Run-time dependency gtkmm-3.0 found: YES 3.24.9
Run-time dependency sigc++-2.0 found: YES 2.12.1
Run-time dependency gstreamer-1.0 found: YES 1.24.10
Run-time dependency gstreamer-plugins-bad-1.0 found: YES 1.24.10
Run-time dependency gstreamer-fft-1.0 found: YES 1.24.10
Run-time dependency lilv-0 found: YES 0.24.24
Run-time dependency libbs2b found: YES 3.1.0
Run-time dependency Boost (found: filesystem, system) found: YES 1.86.0
Run-time dependency sndfile found: YES 1.2.2
Run-time dependency threads found: YES
Library zita-convolver found: YES
Dependency gstreamer-1.0 found: YES 1.24.10 (cached)
Run-time dependency gstreamer-base-1.0 found: YES 1.24.10
Run-time dependency gstreamer-controller-1.0 found: YES 1.24.10
Run-time dependency gstreamer-audio-1.0 found: YES 1.24.10
Dependency sndfile found: YES 1.2.2 (cached)
Run-time dependency samplerate found: YES 0.2.2
Dependency threads found: YES unknown (cached)
Library zita-convolver found: YES
Dependency gstreamer-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-base-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-controller-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-audio-1.0 found: YES 1.24.10 (cached)
Run-time dependency libebur128 found: YES 1.2.6
Dependency gstreamer-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-base-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-controller-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-audio-1.0 found: YES 1.24.10 (cached)
Dependency libebur128 found: YES 1.2.6 (cached)
Dependency gstreamer-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-base-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-controller-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-audio-1.0 found: YES 1.24.10 (cached)
Run-time dependency rnnoise found: YES unknown
Dependency gstreamer-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-base-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-controller-1.0 found: YES 1.24.10 (cached)
Dependency gstreamer-audio-1.0 found: YES 1.24.10 (cached)
Dependency threads found: YES unknown (cached)
Build targets in project: 38

pulseeffects 4.8.7

  User defined options
    auto_features: enabled
    bindir       : /nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/bin
    buildtype    : plain
    includedir   : /nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/include
    infodir      : /nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/share/info
    libdir       : /nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/lib
    libexecdir   : /nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/libexec
    localedir    : /nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/share/locale
    mandir       : /nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/share/man
    prefix       : /nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7
    sbindir      : /nix/store/mismmgirjvcsvxyjqq26rph3s3pqxhww-pulseeffects-4.8.7/sbin
    wrap_mode    : nodownload

Found ninja-1.12.1 at /nix/store/byvgvqlh007flwk940c99zfgzrjg0zsg-ninja-1.12.1/bin/ninja
mesonConfigurePhase: enabled\ parallel\ building
Running phase: buildPhase
build flags: -j12
[1/136] Generating data/pulseeffects-resources_h with a custom command
[2/136] Generating yelp doc help/pt_BR/help-pulseeffects-pt_BR-gmo
[3/136] Generating yelp doc help/ru/help-pulseeffects-ru-gmo
[4/136] Generating yelp doc help/it_IT/help-pulseeffects-it_IT-gmo
[5/136] Generating yelp doc help/de/help-pulseeffects-de-gmo
[6/136] Generating data/pulseeffects-resources_c with a custom command
[7/136] Compiling C object src/pulseeffects.p/meson-generated_.._.._data_pulseeffects-resources.c.o
[8/136] Compiling C++ object src/autogain/libgstpeautogain.so.p/gstpeautogain.cpp.o
[9/136] Compiling C++ object src/crystalizer/libgstpecrystalizer.so.p/.._util.cpp.o
[10/136] Compiling C++ object src/convolver/libgstpeconvolver.so.p/.._util.cpp.o
[11/136] Compiling C++ object src/crystalizer/libgstpecrystalizer.so.p/gstpecrystalizer.cpp.o
[12/136] Compiling C++ object src/crystalizer/libgstpecrystalizer.so.p/filter.cpp.o
[13/136] Compiling C++ object src/convolver/libgstpeconvolver.so.p/gstpeconvolver.cpp.o
[14/136] Compiling C++ object src/pulseeffects.p/spectrum_ui.cpp.o
[15/136] Compiling C++ object src/pulseeffects.p/blocklist_settings_ui.cpp.o
[16/136] Compiling C++ object src/pulseeffects.p/spectrum_preset.cpp.o
[17/136] Compiling C++ object src/pulseeffects.p/pulse_manager.cpp.o
[18/136] Compiling C++ object src/pulseeffects.p/spectrum_settings_ui.cpp.o
[19/136] Compiling C++ object src/pulseeffects.p/pulseeffects.cpp.o
[20/136] Compiling C++ object src/pulseeffects.p/effects_base_ui.cpp.o
[21/136] Compiling C++ object src/pulseeffects.p/app_info_ui.cpp.o
[22/136] Compiling C++ object src/pulseeffects.p/general_settings_ui.cpp.o
[23/136] Compiling C++ object src/pulseeffects.p/pulse_settings_ui.cpp.o
[24/136] Compiling C++ object src/pulseeffects.p/application_ui.cpp.o
[25/136] Compiling C++ object src/pulseeffects.p/application.cpp.o
[26/136] Compiling C++ object src/pulseeffects.p/plugin_base.cpp.o
[27/136] Compiling C++ object src/pulseeffects.p/autogain.cpp.o
[28/136] Compiling C++ object src/pulseeffects.p/sink_input_effects.cpp.o
[29/136] Compiling C++ object src/pulseeffects.p/limiter.cpp.o
[30/136] Compiling C++ object src/pulseeffects.p/source_output_effects.cpp.o
[31/136] Compiling C++ object src/pulseeffects.p/compressor.cpp.o
[32/136] Compiling C++ object src/pulseeffects.p/pipeline_base.cpp.o
[33/136] Compiling C++ object src/pulseeffects.p/multiband_compressor.cpp.o
[34/136] Compiling C++ object src/pulseeffects.p/filter.cpp.o
[35/136] Compiling C++ object src/pulseeffects.p/plugin_ui_base.cpp.o
[36/136] Compiling C++ object src/pulseeffects.p/autogain_ui.cpp.o
[37/136] Compiling C++ object src/pulseeffects.p/equalizer.cpp.o
[38/136] Compiling C++ object src/pulseeffects.p/reverb.cpp.o
[39/136] Compiling C++ object src/pulseeffects.p/limiter_ui.cpp.o
[40/136] Compiling C++ object src/pulseeffects.p/bass_enhancer.cpp.o
[41/136] Compiling C++ object src/pulseeffects.p/pulse_info_ui.cpp.o
[42/136] Compiling C++ object src/pulseeffects.p/compressor_ui.cpp.o
[43/136] Compiling C++ object src/pulseeffects.p/exciter.cpp.o
[44/136] Compiling C++ object src/pulseeffects.p/filter_ui.cpp.o
[45/136] Compiling C++ object src/pulseeffects.p/multiband_compressor_ui.cpp.o
[46/136] Compiling C++ object src/pulseeffects.p/crossfeed.cpp.o
[47/136] Compiling C++ object src/pulseeffects.p/source_output_effects_ui.cpp.o
[48/136] Compiling C++ object src/pulseeffects.p/loudness.cpp.o
[49/136] Compiling C++ object src/pulseeffects.p/reverb_ui.cpp.o
[50/136] Compiling C++ object src/pulseeffects.p/maximizer.cpp.o
[51/136] Compiling C++ object src/pulseeffects.p/bass_enhancer_ui.cpp.o
[52/136] Compiling C++ object src/pulseeffects.p/gate.cpp.o
[53/136] Compiling C++ object src/pulseeffects.p/exciter_ui.cpp.o
[54/136] Compiling C++ object src/pulseeffects.p/sink_input_effects_ui.cpp.o
[55/136] Compiling C++ object src/pulseeffects.p/multiband_gate.cpp.o
[56/136] Compiling C++ object src/pulseeffects.p/stereo_tools.cpp.o
[57/136] Compiling C++ object src/pulseeffects.p/deesser.cpp.o
[58/136] Compiling C++ object src/pulseeffects.p/crossfeed_ui.cpp.o
[59/136] Compiling C++ object src/pulseeffects.p/loudness_ui.cpp.o
[60/136] Compiling C++ object src/pulseeffects.p/convolver.cpp.o
[61/136] Compiling C++ object src/pulseeffects.p/maximizer_ui.cpp.o
[62/136] Compiling C++ object src/pulseeffects.p/pitch.cpp.o
[63/136] Compiling C++ object src/pulseeffects.p/gate_ui.cpp.o
[64/136] Compiling C++ object src/pulseeffects.p/webrtc.cpp.o
[65/136] Compiling C++ object src/pulseeffects.p/crystalizer.cpp.o
[66/136] Compiling C++ object src/pulseeffects.p/delay.cpp.o
[67/136] Compiling C++ object src/pulseeffects.p/stereo_tools_ui.cpp.o
[68/136] Compiling C++ object src/pulseeffects.p/deesser_ui.cpp.o
[69/136] Compiling C++ object src/pulseeffects.p/multiband_gate_ui.cpp.o
[70/136] Compiling C++ object src/pulseeffects.p/rnnoise.cpp.o
[71/136] Compiling C++ object src/pulseeffects.p/convolver_ui.cpp.o
FAILED: src/pulseeffects.p/convolver_ui.cpp.o 
g++ -Isrc/pulseeffects.p -Isrc -I../src -I../include -I. -I.. -Idata -I/nix/store/3j19382v9crsxbdqfy7gaq9i0zvrds2d-pulseaudio-17.0-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/nw1yj7db3rm5g3hajfxdkapxw2r2ychj-glibmm-2.66.7-dev/include/glibmm-2.4 -I/nix/store/zz84m4y3wn5r8nl7qk2jrh6khlk7l629-glibmm-2.66.7/lib/glibmm-2.4/include -I/nix/store/afryl6f0vd3nhdgkakxncpzdqn1qgd82-libsigc++-2.12.1-dev/include/sigc++-2.0 -I/nix/store/zlgmqavnlhnigrsmm2v55p1sj5kpwq3j-libsigc++-2.12.1/lib/sigc++-2.0/include -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/unix-print -I/nix/store/nw1yj7db3rm5g3hajfxdkapxw2r2ychj-glibmm-2.66.7-dev/include/giomm-2.4 -I/nix/store/zz84m4y3wn5r8nl7qk2jrh6khlk7l629-glibmm-2.66.7/lib/giomm-2.4/include -I/nix/store/pz8hhcy9wi57vmkiwbkvfkyfg89ja7zv-gtkmm-3.24.9-dev/include/gtkmm-3.0 -I/nix/store/7fszkhazb1b4i1rgvra6ly5x3c76c3zm-gtkmm-3.24.9/lib/gtkmm-3.0/include -I/nix/store/pz8hhcy9wi57vmkiwbkvfkyfg89ja7zv-gtkmm-3.24.9-dev/include/gdkmm-3.0 -I/nix/store/7fszkhazb1b4i1rgvra6ly5x3c76c3zm-gtkmm-3.24.9/lib/gdkmm-3.0/include -I/nix/store/cn30flwh5zhfxsvpnmsk6zms6fdv7z2m-atkmm-2.28.4-dev/include/atkmm-1.6 -I/nix/store/8lc7f2d7hvyvwaaj4aj0h9z08vqsywic-atkmm-2.28.4/lib/atkmm-1.6/include -I/nix/store/51vl97l2m83i3yb9wr6fxnxjrzlarjnp-cairomm-1.14.5-dev/include/cairomm-1.0 -I/nix/store/gp8jpnrcdqjzi015fp3n97n28d8y7byk-cairomm-1.14.5/lib/cairomm-1.0/include -I/nix/store/qzjfcmn7yxij5b452d1gcpib5k61xkwj-pangomm-2.46.4-dev/include/pangomm-1.4 -I/nix/store/v3n02cp39rnjdkksilmg126bdqd686zf-pangomm-2.46.4/lib/pangomm-1.4/include -I/nix/store/d6zlk1n2sygm961p8viglm8b8nwgbadr-gstreamer-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/a978s37zdcmi466zdqz9kp7izf8pni8s-gst-plugins-bad-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/p1cwigvphrnblsf1cj0nm3rxrr4vri53-gst-plugins-base-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/jij0nas0vz3j4y0546bnb59n8la33c0w-lilv-0.24.24-dev/include/lilv-0 -I/nix/store/sxawd4xh3sg0x4igixib5w290w2axnrb-lv2-1.18.10-dev/include -I/nix/store/qgdvssd7i3402x2azy2rnvrz8f1yzh2s-libbs2b-3.1.0/include/bs2b -I/nix/store/f0qbspim8mzycf2df88sqvvmfnd7yxxj-boost-1.86.0-dev/include -I/nix/store/vr9k0553dgkpr2ckjbsia9k1ck6z84wd-libsndfile-1.2.2-dev/include -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++17 '-DG_LOG_DOMAIN="pulseeffects"' -pthread -DBOOST_SYSTEM_DYN_LINK=1 -DBOOST_FILESYSTEM_DYN_LINK=1 -DBOOST_ALL_NO_LIB -D_REENTRANT -MD -MQ src/pulseeffects.p/convolver_ui.cpp.o -MF src/pulseeffects.p/convolver_ui.cpp.o.d -o src/pulseeffects.p/convolver_ui.cpp.o -c ../src/convolver_ui.cpp
../src/convolver_ui.cpp: In member function 'void ConvolverUi::import_irs_file(const std::string&)':
../src/convolver_ui.cpp:189:66: error: 'boost::filesystem::copy_option' has not been declared
  189 |     boost::filesystem::copy_file(p, out_path, boost::filesystem::copy_option::overwrite_if_exists);
      |                                                                  ^~~~~~~~~~~
[72/136] Compiling C++ object src/pulseeffects.p/pitch_ui.cpp.o
[73/136] Compiling C++ object src/pulseeffects.p/webrtc_ui.cpp.o
[74/136] Compiling C++ object src/pulseeffects.p/equalizer_ui.cpp.o
[75/136] Compiling C++ object src/pulseeffects.p/delay_preset.cpp.o
[76/136] Compiling C++ object src/pulseeffects.p/delay_ui.cpp.o
[77/136] Compiling C++ object src/pulseeffects.p/crystalizer_ui.cpp.o
[78/136] Compiling C++ object src/pulseeffects.p/autogain_preset.cpp.o
[79/136] Compiling C++ object src/pulseeffects.p/presets_manager.cpp.o
FAILED: src/pulseeffects.p/presets_manager.cpp.o 
g++ -Isrc/pulseeffects.p -Isrc -I../src -I../include -I. -I.. -Idata -I/nix/store/3j19382v9crsxbdqfy7gaq9i0zvrds2d-pulseaudio-17.0-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/nw1yj7db3rm5g3hajfxdkapxw2r2ychj-glibmm-2.66.7-dev/include/glibmm-2.4 -I/nix/store/zz84m4y3wn5r8nl7qk2jrh6khlk7l629-glibmm-2.66.7/lib/glibmm-2.4/include -I/nix/store/afryl6f0vd3nhdgkakxncpzdqn1qgd82-libsigc++-2.12.1-dev/include/sigc++-2.0 -I/nix/store/zlgmqavnlhnigrsmm2v55p1sj5kpwq3j-libsigc++-2.12.1/lib/sigc++-2.0/include -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/unix-print -I/nix/store/nw1yj7db3rm5g3hajfxdkapxw2r2ychj-glibmm-2.66.7-dev/include/giomm-2.4 -I/nix/store/zz84m4y3wn5r8nl7qk2jrh6khlk7l629-glibmm-2.66.7/lib/giomm-2.4/include -I/nix/store/pz8hhcy9wi57vmkiwbkvfkyfg89ja7zv-gtkmm-3.24.9-dev/include/gtkmm-3.0 -I/nix/store/7fszkhazb1b4i1rgvra6ly5x3c76c3zm-gtkmm-3.24.9/lib/gtkmm-3.0/include -I/nix/store/pz8hhcy9wi57vmkiwbkvfkyfg89ja7zv-gtkmm-3.24.9-dev/include/gdkmm-3.0 -I/nix/store/7fszkhazb1b4i1rgvra6ly5x3c76c3zm-gtkmm-3.24.9/lib/gdkmm-3.0/include -I/nix/store/cn30flwh5zhfxsvpnmsk6zms6fdv7z2m-atkmm-2.28.4-dev/include/atkmm-1.6 -I/nix/store/8lc7f2d7hvyvwaaj4aj0h9z08vqsywic-atkmm-2.28.4/lib/atkmm-1.6/include -I/nix/store/51vl97l2m83i3yb9wr6fxnxjrzlarjnp-cairomm-1.14.5-dev/include/cairomm-1.0 -I/nix/store/gp8jpnrcdqjzi015fp3n97n28d8y7byk-cairomm-1.14.5/lib/cairomm-1.0/include -I/nix/store/qzjfcmn7yxij5b452d1gcpib5k61xkwj-pangomm-2.46.4-dev/include/pangomm-1.4 -I/nix/store/v3n02cp39rnjdkksilmg126bdqd686zf-pangomm-2.46.4/lib/pangomm-1.4/include -I/nix/store/d6zlk1n2sygm961p8viglm8b8nwgbadr-gstreamer-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/a978s37zdcmi466zdqz9kp7izf8pni8s-gst-plugins-bad-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/p1cwigvphrnblsf1cj0nm3rxrr4vri53-gst-plugins-base-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/jij0nas0vz3j4y0546bnb59n8la33c0w-lilv-0.24.24-dev/include/lilv-0 -I/nix/store/sxawd4xh3sg0x4igixib5w290w2axnrb-lv2-1.18.10-dev/include -I/nix/store/qgdvssd7i3402x2azy2rnvrz8f1yzh2s-libbs2b-3.1.0/include/bs2b -I/nix/store/f0qbspim8mzycf2df88sqvvmfnd7yxxj-boost-1.86.0-dev/include -I/nix/store/vr9k0553dgkpr2ckjbsia9k1ck6z84wd-libsndfile-1.2.2-dev/include -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++17 '-DG_LOG_DOMAIN="pulseeffects"' -pthread -DBOOST_SYSTEM_DYN_LINK=1 -DBOOST_FILESYSTEM_DYN_LINK=1 -DBOOST_ALL_NO_LIB -D_REENTRANT -MD -MQ src/pulseeffects.p/presets_manager.cpp.o -MF src/pulseeffects.p/presets_manager.cpp.o.d -o src/pulseeffects.p/presets_manager.cpp.o -c ../src/presets_manager.cpp
../src/presets_manager.cpp: In member function 'void PresetsManager::import(PresetType, const std::string&)':
../src/presets_manager.cpp:475:68: error: 'boost::filesystem::copy_option' has not been declared
  475 |       boost::filesystem::copy_file(p, out_path, boost::filesystem::copy_option::overwrite_if_exists);
      |                                                                    ^~~~~~~~~~~
[80/136] Compiling C++ object src/pulseeffects.p/limiter_preset.cpp.o
[81/136] Compiling C++ object src/pulseeffects.p/rnnoise_ui.cpp.o
[82/136] Compiling C++ object src/pulseeffects.p/presets_menu_ui.cpp.o
ninja: build stopped: subcommand failed.
error: builder for '/nix/store/1w34lhgb0fqdfwj01lmjaq8smm0gpsa9-pulseeffects-4.8.7.drv' failed with exit code 1;
       last 25 log lines:
       > [71/136] Compiling C++ object src/pulseeffects.p/convolver_ui.cpp.o
       > FAILED: src/pulseeffects.p/convolver_ui.cpp.o
       > g++ -Isrc/pulseeffects.p -Isrc -I../src -I../include -I. -I.. -Idata -I/nix/store/3j19382v9crsxbdqfy7gaq9i0zvrds2d-pulseaudio-17.0-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/nw1yj7db3rm5g3hajfxdkapxw2r2ychj-glibmm-2.66.7-dev/include/glibmm-2.4 -I/nix/store/zz84m4y3wn5r8nl7qk2jrh6khlk7l629-glibmm-2.66.7/lib/glibmm-2.4/include -I/nix/store/afryl6f0vd3nhdgkakxncpzdqn1qgd82-libsigc++-2.12.1-dev/include/sigc++-2.0 -I/nix/store/zlgmqavnlhnigrsmm2v55p1sj5kpwq3j-libsigc++-2.12.1/lib/sigc++-2.0/include -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/unix-print -I/nix/store/nw1yj7db3rm5g3hajfxdkapxw2r2ychj-glibmm-2.66.7-dev/include/giomm-2.4 -I/nix/store/zz84m4y3wn5r8nl7qk2jrh6khlk7l629-glibmm-2.66.7/lib/giomm-2.4/include -I/nix/store/pz8hhcy9wi57vmkiwbkvfkyfg89ja7zv-gtkmm-3.24.9-dev/include/gtkmm-3.0 -I/nix/store/7fszkhazb1b4i1rgvra6ly5x3c76c3zm-gtkmm-3.24.9/lib/gtkmm-3.0/include -I/nix/store/pz8hhcy9wi57vmkiwbkvfkyfg89ja7zv-gtkmm-3.24.9-dev/include/gdkmm-3.0 -I/nix/store/7fszkhazb1b4i1rgvra6ly5x3c76c3zm-gtkmm-3.24.9/lib/gdkmm-3.0/include -I/nix/store/cn30flwh5zhfxsvpnmsk6zms6fdv7z2m-atkmm-2.28.4-dev/include/atkmm-1.6 -I/nix/store/8lc7f2d7hvyvwaaj4aj0h9z08vqsywic-atkmm-2.28.4/lib/atkmm-1.6/include -I/nix/store/51vl97l2m83i3yb9wr6fxnxjrzlarjnp-cairomm-1.14.5-dev/include/cairomm-1.0 -I/nix/store/gp8jpnrcdqjzi015fp3n97n28d8y7byk-cairomm-1.14.5/lib/cairomm-1.0/include -I/nix/store/qzjfcmn7yxij5b452d1gcpib5k61xkwj-pangomm-2.46.4-dev/include/pangomm-1.4 -I/nix/store/v3n02cp39rnjdkksilmg126bdqd686zf-pangomm-2.46.4/lib/pangomm-1.4/include -I/nix/store/d6zlk1n2sygm961p8viglm8b8nwgbadr-gstreamer-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/a978s37zdcmi466zdqz9kp7izf8pni8s-gst-plugins-bad-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/p1cwigvphrnblsf1cj0nm3rxrr4vri53-gst-plugins-base-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/jij0nas0vz3j4y0546bnb59n8la33c0w-lilv-0.24.24-dev/include/lilv-0 -I/nix/store/sxawd4xh3sg0x4igixib5w290w2axnrb-lv2-1.18.10-dev/include -I/nix/store/qgdvssd7i3402x2azy2rnvrz8f1yzh2s-libbs2b-3.1.0/include/bs2b -I/nix/store/f0qbspim8mzycf2df88sqvvmfnd7yxxj-boost-1.86.0-dev/include -I/nix/store/vr9k0553dgkpr2ckjbsia9k1ck6z84wd-libsndfile-1.2.2-dev/include -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++17 '-DG_LOG_DOMAIN="pulseeffects"' -pthread -DBOOST_SYSTEM_DYN_LINK=1 -DBOOST_FILESYSTEM_DYN_LINK=1 -DBOOST_ALL_NO_LIB -D_REENTRANT -MD -MQ src/pulseeffects.p/convolver_ui.cpp.o -MF src/pulseeffects.p/convolver_ui.cpp.o.d -o src/pulseeffects.p/convolver_ui.cpp.o -c ../src/convolver_ui.cpp
       > ../src/convolver_ui.cpp: In member function 'void ConvolverUi::import_irs_file(const std::string&)':
       > ../src/convolver_ui.cpp:189:66: error: 'boost::filesystem::copy_option' has not been declared
       >   189 |     boost::filesystem::copy_file(p, out_path, boost::filesystem::copy_option::overwrite_if_exists);
       >       |                                                                  ^~~~~~~~~~~
       > [72/136] Compiling C++ object src/pulseeffects.p/pitch_ui.cpp.o
       > [73/136] Compiling C++ object src/pulseeffects.p/webrtc_ui.cpp.o
       > [74/136] Compiling C++ object src/pulseeffects.p/equalizer_ui.cpp.o
       > [75/136] Compiling C++ object src/pulseeffects.p/delay_preset.cpp.o
       > [76/136] Compiling C++ object src/pulseeffects.p/delay_ui.cpp.o
       > [77/136] Compiling C++ object src/pulseeffects.p/crystalizer_ui.cpp.o
       > [78/136] Compiling C++ object src/pulseeffects.p/autogain_preset.cpp.o
       > [79/136] Compiling C++ object src/pulseeffects.p/presets_manager.cpp.o
       > FAILED: src/pulseeffects.p/presets_manager.cpp.o
       > g++ -Isrc/pulseeffects.p -Isrc -I../src -I../include -I. -I.. -Idata -I/nix/store/3j19382v9crsxbdqfy7gaq9i0zvrds2d-pulseaudio-17.0-dev/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0 -I/nix/store/rb870mk51kxlxq9crkldzs8n4vwr9sk4-glib-2.82.1/lib/glib-2.0/include -I/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include -I/nix/store/nw1yj7db3rm5g3hajfxdkapxw2r2ychj-glibmm-2.66.7-dev/include/glibmm-2.4 -I/nix/store/zz84m4y3wn5r8nl7qk2jrh6khlk7l629-glibmm-2.66.7/lib/glibmm-2.4/include -I/nix/store/afryl6f0vd3nhdgkakxncpzdqn1qgd82-libsigc++-2.12.1-dev/include/sigc++-2.0 -I/nix/store/zlgmqavnlhnigrsmm2v55p1sj5kpwq3j-libsigc++-2.12.1/lib/sigc++-2.0/include -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0 -I/nix/store/zviv60hd5v5qggfbl0n20nwlkzzvvyyq-at-spi2-core-2.54.0-dev/include/atk-1.0 -I/nix/store/l0np4qv5260igzj1z5cz4j936i29bn1j-cairo-1.18.2-dev/include/cairo -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include/freetype2 -I/nix/store/wc97xp20zf23vx6rj65ah6knqpqwn9bn-freetype-2.13.3-dev/include -I/nix/store/v5flgachjhgvsxqxqi8hnpi19cnbgfwc-gdk-pixbuf-2.42.12-dev/include/gdk-pixbuf-2.0 -I/nix/store/akxwqfa5h9n4giv2zp04m3v4ngabb18j-pango-1.54.0-dev/include/pango-1.0 -I/nix/store/9w2ars9rx8k0y48pq9fbmqxypmmfga3q-harfbuzz-10.1.0-dev/include/harfbuzz -I/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/unix-print -I/nix/store/nw1yj7db3rm5g3hajfxdkapxw2r2ychj-glibmm-2.66.7-dev/include/giomm-2.4 -I/nix/store/zz84m4y3wn5r8nl7qk2jrh6khlk7l629-glibmm-2.66.7/lib/giomm-2.4/include -I/nix/store/pz8hhcy9wi57vmkiwbkvfkyfg89ja7zv-gtkmm-3.24.9-dev/include/gtkmm-3.0 -I/nix/store/7fszkhazb1b4i1rgvra6ly5x3c76c3zm-gtkmm-3.24.9/lib/gtkmm-3.0/include -I/nix/store/pz8hhcy9wi57vmkiwbkvfkyfg89ja7zv-gtkmm-3.24.9-dev/include/gdkmm-3.0 -I/nix/store/7fszkhazb1b4i1rgvra6ly5x3c76c3zm-gtkmm-3.24.9/lib/gdkmm-3.0/include -I/nix/store/cn30flwh5zhfxsvpnmsk6zms6fdv7z2m-atkmm-2.28.4-dev/include/atkmm-1.6 -I/nix/store/8lc7f2d7hvyvwaaj4aj0h9z08vqsywic-atkmm-2.28.4/lib/atkmm-1.6/include -I/nix/store/51vl97l2m83i3yb9wr6fxnxjrzlarjnp-cairomm-1.14.5-dev/include/cairomm-1.0 -I/nix/store/gp8jpnrcdqjzi015fp3n97n28d8y7byk-cairomm-1.14.5/lib/cairomm-1.0/include -I/nix/store/qzjfcmn7yxij5b452d1gcpib5k61xkwj-pangomm-2.46.4-dev/include/pangomm-1.4 -I/nix/store/v3n02cp39rnjdkksilmg126bdqd686zf-pangomm-2.46.4/lib/pangomm-1.4/include -I/nix/store/d6zlk1n2sygm961p8viglm8b8nwgbadr-gstreamer-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/a978s37zdcmi466zdqz9kp7izf8pni8s-gst-plugins-bad-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/p1cwigvphrnblsf1cj0nm3rxrr4vri53-gst-plugins-base-1.24.10-dev/include/gstreamer-1.0 -I/nix/store/jij0nas0vz3j4y0546bnb59n8la33c0w-lilv-0.24.24-dev/include/lilv-0 -I/nix/store/sxawd4xh3sg0x4igixib5w290w2axnrb-lv2-1.18.10-dev/include -I/nix/store/qgdvssd7i3402x2azy2rnvrz8f1yzh2s-libbs2b-3.1.0/include/bs2b -I/nix/store/f0qbspim8mzycf2df88sqvvmfnd7yxxj-boost-1.86.0-dev/include -I/nix/store/vr9k0553dgkpr2ckjbsia9k1ck6z84wd-libsndfile-1.2.2-dev/include -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++17 '-DG_LOG_DOMAIN="pulseeffects"' -pthread -DBOOST_SYSTEM_DYN_LINK=1 -DBOOST_FILESYSTEM_DYN_LINK=1 -DBOOST_ALL_NO_LIB -D_REENTRANT -MD -MQ src/pulseeffects.p/presets_manager.cpp.o -MF src/pulseeffects.p/presets_manager.cpp.o.d -o src/pulseeffects.p/presets_manager.cpp.o -c ../src/presets_manager.cpp
       > ../src/presets_manager.cpp: In member function 'void PresetsManager::import(PresetType, const std::string&)':
       > ../src/presets_manager.cpp:475:68: error: 'boost::filesystem::copy_option' has not been declared
       >   475 |       boost::filesystem::copy_file(p, out_path, boost::filesystem::copy_option::overwrite_if_exists);
       >       |                                                                    ^~~~~~~~~~~
       > [80/136] Compiling C++ object src/pulseeffects.p/limiter_preset.cpp.o
       > [81/136] Compiling C++ object src/pulseeffects.p/rnnoise_ui.cpp.o
       > [82/136] Compiling C++ object src/pulseeffects.p/presets_menu_ui.cpp.o
       > ninja: build stopped: subcommand failed.
```

</details>

## Additional context

I have been able to build this package without any issue a few weeks ago,
and since the last relevant `package.nix` modification is 9 month old, the actual issue certainly lies somewhere else.

## Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.63, NixOS, 25.05 (Warbler), 25.05.20250104.8f3e1f8`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.11`
 - channels(root): `"home-manager, nixos"`
 - nixpkgs: `/nix/store/khbvilmsrv4l69nwd52h27j1mp44a0xi-source`

## Notify maintainers

No one in the `meta.maintainers` list.

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/pu/pulseeffects-legacy/package.nix
+++ b/pkgs/by-name/pu/pulseeffects-legacy/package.nix
@@ -46,15 +46,15 @@ let
     zam-plugins # maximizer
   ];
 in
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "pulseeffects";
-  version = "4.8.7";
+  version = "4.8.7-unstable-2024-09-17";
 
   src = fetchFromGitHub {
     owner = "wwmm";
     repo = "pulseeffects";
-    rev = "v${version}";
-    sha256 = "sha256-ldvcA8aTHOgaascH6MF4CzmJ8I2rYOiR0eAkCZzvK/M=";
+    rev = "fbe0a724c1405cee624802f381476cf003dfcfa";
+    hash = "sha256-tyVUWc8w08WUnJRTjJVTIiG/YBWTETNYG+4amwEYezY=";
   };
 
   nativeBuildInputs = [

